### ISSUE TITLE
Put a checkbox to each block for bulk deletion

### ISSUE THREAD

**Issue by La-Geek** [WordPress/gutenberg#12022]
For users who don't know HTML (and so also can't work with HTML editor) it will be very uncomfortable to delete several blocks.

I had duplicated a paragraph block for test purposes (testing an issue here) 20 times. Today I wanted to use the same post and delete the blocks. Wow! Click into every block, then click on the 3 vertical points and then finally! click on "remove this block". Click 3 x 20 times ....

Yes, I am able to deselect all blocks, click on the vertical points on top right, activate "code editor" and to delete the unwanted content, and keep the needed comments intact.

But what is about the at a guess 29,889.000 users who don't know?

**Comment by GlennMartin1** [comment]
An alternative method I use is to select a range of blocks, then click the DELETE key.

**Comment by La-Geek** [comment]
Yes. Shift and click did it.

What I tried was CTRL and Click, what does in fact do not work.

**Comment by La-Geek** [comment]
No I tried CTRL/STRG and Click, this does not work

With Click on the first block and holding Shift-Key and Click on the last block I can select multiple blocks, which are spatially connected.

What I tried too, and what did not work yesterday, but today it does (?): Set the cursor at the beginning of a block and draw with the mouse to the endpoint and all is selected and I can hit the Delete-Key.

However, it would be an improvement to have checkboxes like you have in Plugins or Update overview. And this is not a bug report, it's an improvement suggestion :)
